Marlin/Marlin_main.cpp

```diff
@@ -9421,6 +9576,18 @@ void setup() {
   setup_photpin();
   servo_init();
 
+ #if ENABLED(SPINDLE_ENABLE)
+     OUT_WRITE(SPINDLE_ENABLE_PIN, !SPINDLE_ENABLE_INVERT);  //init spindle to off;
+     #if ENABLED(SPINDLE_SPEED)
+       pinMode(SPINDLE_SPEED_PIN, OUTPUT);
+       #if SPINDLE_SPEED_INVERT == false        // init PWM to lowset speed
```

Comment:
Since you've chosen to make `SPINDLE_SPEED_INVERT` a `true` or `false` value you should use it with that in mind. So instead of conditional block here, use it like this instead:
```cpp
analogWrite(SPINDLE_SPEED_PIN, SPINDLE_SPEED_INVERT ? 255 : 0);
```